modules/recast/register_types.cpp

```diff
@@ -40,7 +40,14 @@ void register_recast_types() {
 #ifdef TOOLS_ENABLED
 	EditorPlugins::add_by_type<NavigationMeshEditorPlugin>();
 	_nav_mesh_generator = memnew(EditorNavigationMeshGenerator);
+
+	ClassDB::APIType prev_api = ClassDB::get_current_api();
+	ClassDB::set_current_api(ClassDB::API_EDITOR);
```

Comment:
Shouldn't we have a `ClassDB::register_editor_class| instead to make this logic less error prone?

Reply:
I don't have any objections.